client/src/components/SliderCard.jsx

```diff
@@ -31,10 +65,12 @@ class SliderCard extends React.Component {
     return (
       <>
       <StyledCard>
-        <img src={this.props.image[0]}></img>
-        <StyledText>{this.props.title}</StyledText>
-        <StyledText>{this.props.inventory.price}</StyledText>
-        <StyledText>icons</StyledText>
+        <CardPictureStyle >
```

Comment:
could use de-structuring to make it less wordy.

Reply:
Yes, this will change when I start incorporating live data from the API calls it makes on the back end